content/05.conclusions.md

```diff
@@ -1,6 +1,59 @@
+Even parts of the text that the tool struggle to revise proved to be useful for the authors to identify paragraphs that would challenge human readers.
+
+
+We designed section-specific prompts to guide the AI-based revision of the text in a way that is consistent with the section.
+To test this, we used three manuscripts across all the standard sections of an article in the life sciences.
+These sections represented a wide range of complexity levels for a revision task: from abstracts that need to briefly describe the research questions and proposed solutions, to the Methods section that focuses in very specific technical details, sometimes needing a particular format (such as equations).
+Overall, the tool provided very good suggestions that significantly improved the sentence structure.
+The models were able to capture the main ideas in the original text and generate a revision that was easier to read and follow, yet communicating the original meaning more clearly and concisely.
+In paragraphs from the Methods section, the model even understood the mathematical symbols in an equation and fixed references to them.
+We found that the most challenging section to revise was the abstract, which basically needs to describe the state-of-the-art in a field, the importance of the research questions, and briefly communicate how the proposed methods or analyses address them.
+This could be addressed either by designing better prompts (using a "few-shot learning" approach [@doi:10.1145/3386252]), which would be the fastest way, or by fine-tuning the pre-trained models using a large corpus of abstracts, which would potentially produce higher-quality results.
+We believe that the AI-based revision tool, although with some errors and minor issues, performed very well on the rest of the sections.
+As showed in the use cases, the improvements suggested by the model surpassed the mistakes that it eventually made, which can be easily fixed by a human after reviewing the suggestions.
+It is important to note, however, that our assessment of the quality of the revisions was necessarily subjective, as there could be, for example, writing styles that might not be widely shared across researchers.
+We tried to overcome this by selecting manuscripts of our own to more objectively assess the revisions, but we acknowledge that this is an important limitation of our work.
+Only time and a more extensive use of the tool will tell if an automated, AI-based revision approach for scientific literature will become a standard practice in the future.
+
+
+Our AI-based tool uses a paragraph-by-paragraph approach to revision, meaning that the model is applied to each paragraph of the manuscript *independently*, without considering the context of the paragraph.
+In addition to producing acceptable results, this divide-and-conquer approach has advantages, such as the ability to process long texts without strongly depending context length of the models.
```

Comment:
I don't fully understand this - can you clarify? You're saying this can work with shorter queries so you can use more than just the most complex models?